example/offload-template/main.cc

```diff
@@ -0,0 +1,43 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file example/offload-template/main.cc
+//! \brief Minimal Geant4 application with Celeritas offloading.
+//---------------------------------------------------------------------------//
+#include <FTFP_BERT.hh>
+#include <G4RunManager.hh>
+
+#include "src/ActionInitialization.hh"
+#include "src/DetectorConstruction.hh"
+
+//---------------------------------------------------------------------------//
+/*!
+ * Geant4-Celeritas offloading template.
+ *
+ * See README for details.
+ */
+int main(int argc, char* argv[])
+{
+    if (argc != 1)
+    {
+        // Print help message
+        std::cout << "Usage: " << argv[0] << std::endl;
+        return EXIT_FAILURE;
+    }
+
+    // Construct run manager
+    G4RunManager run_manager;
```

Comment:
```suggestion
      auto runManager = G4RunManagerFactory::CreateRunManager(G4RunManagerType::Default);
```

Reply:
Would this make it MT if Geant4 is built with threading?

Reply:
I actually made this a `G4RunManagerType::Serial` to run a single-thread execution, and just added a comment on the MT case.

Reply:
GEANT4_BUILD_MULTITHREADED=ON is now the default (so, you do not need explicitly to set it), which then is able to run both sequential and tasking (default tasking option is the traditional old MT).  So it will support all cases.